Services/AssessmentQuestion/interfaces/interface.ilAsqQuestionPresentation.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/* Copyright (c) 1998-2013 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * Interface ilAsqQuestionPresentation
+ *
+ * @author    Björn Heyser <info@bjoernheyser.de>
+ * @version    $Id$
+ *
+ * @package    Services/AssessmentQuestion
+ */
+interface ilAsqQuestionPresentation
+{
+	/**
+	 * @param ilAsqQuestion $question
+	 * @return void
+	 */
+	public function setQuestion(ilAsqQuestion $question);
```

Comment:
I am not sure if I understand this, do we have exactly one Presentation for each question type?

Reply:
yes, furthermore, there are different presenations depending on a rendering purpose, i will follow your advice to have different presentation classes for each rendering purpose and question type. So the existing presentation classes will be the number of question types multiplicated with separate rendering purposes.